src/udev/udev-manager.c

```diff
@@ -262,22 +262,25 @@ static void manager_reload(Manager *manager, bool force) {
         /* Reload SELinux label database, to make the child inherit the up-to-date database. */
         mac_selinux_maybe_reload();
 
-        /* Nothing changed. It is not necessary to reload. */
-        if (!udev_rules_should_reload(manager->rules) && !udev_builtin_should_reload()) {
-
+        UdevReloadFlag flags = udev_builtin_should_reload();
```

Comment:
the variable is named in plural, the type in singular. I'd probably adjust the type to also be in plural.